schematools/contrib/django/models.py

```diff
@@ -130,12 +130,12 @@ def get_dataset(cls) -> DatasetSchema:
 
     @classmethod
     def get_dataset_id(cls) -> str:
-        return cls._table_schema._parent_schema.id
+        return slugify(cls._table_schema._parent_schema.id, sign="_")
```

Comment:
ik snap dat speciale tekens onwenselijk is, maar vraag me af of dit de plek is om het te corrigeren..

Reply:
Hmm. Ik had dit eerste in DSO API gecorrigeerd (bij de aanroep). Nick gaf toen aan dat de spaties ook op andere plekken tot problemen zouden kunnen leiden. Daar vond ik wat in zitten, dus is het verplaatst naar de schema tools. Maar, ben met je eens dat het een beetje voelt als een "side-effect".